### PR TITLE
Bed 4752

### PR DESCRIPTION
**Pull request by stephanieslamb**
## Description

Updated specified log errors to warns in order to cut down noise.  

## Motivation and Context

This PR addresses: BED-4752

*Why is this change required? What problem does it solve?*
This change was requested to lessen confusion post releases or while investigating certain issues. 

## How Has This Been Tested?
No real code changes.  Unit tests still pass. 

## Types of changes

- Chore (a change that does not modify the application functionality)

## Checklist:

- [X] I have met the contributing prerequisites
  - Assigned myself to this PR
  - Added the appropriate labels
  - Associated an issue: https://github.com/SpecterOps/BloodHound/issues/672
  - Read the Contributing guide: https://github.com/SpecterOps/BloodHound/wiki/Contributing
- [X] I have ensured that related documentation is up-to-date
  - Open API docs
  - Code comments (GoDocs / JSDocs)
- [X] I have followed proper test practices
  - Added/updated tests to cover my changes
  - All new and existing tests passed

### CODE REVIEW

File: packages/go/analysis/ad/esc3.go [hunk 1]
```diff
@@ -140,7 +141,9 @@ func PostEnrollOnBehalfOf(certTemplates []*graph.Node, operation analysis.StatTr
 	versionTwoTemplates := make([]*graph.Node, 0)
 
 	for _, node := range certTemplates {
-		if version, err := node.Properties.Get(ad.SchemaVersion.String()).Float64(); err != nil {
+		if version, err := node.Properties.Get(ad.SchemaVersion.String()).Float64(); errors.Is(err, graph.ErrPropertyNotFound) {
+			log.Warnf("Did not get schema version for cert template %d: %v", node.ID, err)
```
Comment (mvlipka) [inline packages/go/analysis/ad/esc3.go:145]: Nit: mixing `Did not` and `Didnt` can make it confusing to search logs

File: cmd/api/src/daemons/datapipe/jobs.go [hunk 2]
```diff
@@ -214,7 +216,9 @@ func (s *Daemon) processIngestFile(ctx context.Context, path string, fileType mo
 
 				if err := file.Close(); err != nil {
 					log.Errorf("Error closing ingest file %s: %v", filePath, err)
-				} else if err := os.Remove(filePath); err != nil {
+				} else if err := os.Remove(filePath); errors.Is(err, fs.ErrNotExist) {
+					log.Warnf("Removing ingest file %s: %v", filePath, err)
```
Comment (mvlipka) [inline cmd/api/src/daemons/datapipe/jobs.go:220]: Nit: `%w` probably makes more sense than `%v`.
While they are synonymous, `%w` enforces the `Error` interface to be implemented, which for a lot of the warnings we have, it seems like that's the case that we want.

I see `%v` was already being used, and we use it a lot elsewhere, so moreso just an observation

File: packages/go/ein/azure.go [hunk 3]
```diff
@@ -41,7 +41,10 @@ const (
 	KeyVaultPermissionGet string = "Get"
 )
 
-var resourceGroupLevel = regexp.MustCompile(`^[\\w\\d\\-\\/]*/resourceGroups/[0-9a-zA-Z]+$`)
+var (
+	resourceGroupLevel = regexp.MustCompile(`^[\\w\\d\\-\\/]*/resourceGroups/[0-9a-zA-Z]+$`)
+	InvalidTypeErr     = errors.New("invalid type returned from directory object")
```
Comment (mvlipka) [inline packages/go/ein/azure.go:46]: Great refactor!

**Review by mvlipka — APPROVED** [review]
LGTM
Couple of nits, but, it doesn't appear we really have as much of a standard when it comes to logging warnings.
Errors are generally prefixed with one of [`Error`, 'Failed', 'Unabled'], but we don't seem to have as much consensus when it comes to warn level messages.

File: cmd/api/src/daemons/datapipe/jobs.go (hunk 2 above)
Comment (stephanieslamb) [reply cmd/api/src/daemons/datapipe/jobs.go]: thanks for the explanation!  went ahead and updated.

**Review by mvlipka — APPROVED** [review]

File: cmd/api/src/queries/graph.go [hunk 4]
```diff
@@ -21,6 +21,7 @@ package queries
 import (
 	"bytes"
 	"context"
+	e "errors"
```
Comment (zinic) [inline cmd/api/src/queries/graph.go:24]: This is a sign that we should refactor away from using our (admittedly) ancient internal `errors` package. Please update this file to remove the use of `github.com/specterops/bloodhound/errors`.

**Review by zinic — CHANGES_REQUESTED** [review]
One request for refactor, changeset otherwise looks solid

File: cmd/api/src/queries/graph.go (hunk 4 above)
Comment (stephanieslamb) [reply cmd/api/src/queries/graph.go]: updated file to use `errors`

**Review by zinic — APPROVED** [review]

**Pull request merged**